### ISSUE TITLE
Enhance eraser function to remove only drawing above an inserted images

### ISSUE THREAD

**Issue by Leiyi548** [zsviczian/obsidian-excalidraw-plugin#2047]
### Is your feature request related to a problem? Please describe.

I am insert an image in obsidian-excalidraw, and i am doing some extra drawing in the images,
when i make mistake and want to use eraser to erase some part of my drawing , it will also erase my inserted image, but i want to keep my insert image, only erase the drawing.

### Describe the solution you'd like

 i want to keep my insert image, only erase the drawing.

**Comment by zsviczian** [comment]
agree, this would be a great feature. Please raise it on https://github.com/excalidraw/excalidraw/issues.

In the meantime you can lock the image then use the eraser, then unlock the image, or simply keep the images locked. This workaround could easily be converted into an Excalidraw script with a shortcut. 1) find images in the drawing, 2) set them to locked, 3) enable eraser. Another script could unlock all your images. I understand not ideal, but until the feature is implemented in Excalidraw.com, this is the workaround that came to mind...

**Comment by Leiyi548** [comment]
> Excalidraw

@zsviczian Thank you your answer.
How to lock image in obsidian excalidraw?I not find this option.

**Comment by zsviczian** [comment]
Right click on the image and select lock
